### PR TITLE
[Delivers #149494485] Filter Service - Action - Filter Photo

### PR DESCRIPTION
**Pull request by jonajonlee**
- added Photo-Filter microservice
- changed Photo-Storage Upload endpoint to take an image buffer

s3photos-greyscalesample

### CODE REVIEW

File: apps/photo-filter/server.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const app = require('express')();
```
Comment (Harrison-M) [inline apps/photo-filter/server.js:14]: Needs license

File: apps/photo-filter/server.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// Endpoint: Apply greyscale filter to an image
+app.post('/greyscale', require('./routes/greyscale'));
+
+// catch all if a path does not exist
+app.use((req, res) => {
+  res.status(404).json({ code: 'RouteNotFound' });
+});
+
+app.listen(PORT, () => {
+  console.log(`Photo Storage API listening on http://localhost:${PORT}`);
```
Comment (Harrison-M) [inline apps/photo-filter/server.js]: s/Storage/Filter

**Review by Harrison-M — CHANGES_REQUESTED** [review]
These two things plus the gif error

**Review by sreimer15 — APPROVED** [review]
Besides Gif Issue it looks good to me

File: apps/photo-filter/server.js (hunk 1 above)
Comment (jonajonlee) [reply apps/photo-filter/server.js:14]: fixed

**Review by Harrison-M — APPROVED** [review]

**Pull request merged**